src/meta/user/userRoles.ts

```diff
@@ -1,5 +1,16 @@
+import { RoleName } from './userRole'
+
 const noRole = { role: 'NONE', labelKey: 'user.roles.noRole' }
 
+const roles = [
```

Comment:
where is this used? is it really necessary ?

Reply:
call it roleNames if you really need it. I would prefer not to have it

Reply:
renamed into roleNames. It is used by two different components: UserListHeader and AdministrationListElement